pkgs/top-level/python-packages.nix

```
@@ -3707,6 +3707,8 @@ self: super: with self; {
 
   fastcache = callPackage ../development/python-modules/fastcache { };
 
+  fastchat = callPackage ../development/python-modules/fastchat { };
```

```suggestion
  fschat = callPackage ../development/python-modules/fschat { };
```
I think we should rename an attribute name and a file name, too.